### PR TITLE
cycle trough images

### PR DESCRIPTION
**Pull request by Much4cho**

### CODE REVIEW

**Review by aszulc97 — CHANGES_REQUESTED** [review]
jezus maria

**Comment by Much4cho** [comment]
> jezus maria

imglish plz

**Comment by aszulc97** [comment]
Jesús María

File: src/app/main-page/main-page.component.ts [hunk 1]
```diff
@@ -16,8 +20,31 @@ export class MainPageComponent implements OnInit {
     this.prepareListOfImages();
   }
 
-  prepareListOfImages(){
-   // TODO
-   // this images = ...
+  prepareListOfImages(): void{
```
Comment (aszulc97) [inline src/app/main-page/main-page.component.ts:23]: The plan was to make it in a dynamic way, so whenever user adds a new pic to a 'img' folder, the code is going to update our site xoxo

**Review by aszulc97 — CHANGES_REQUESTED** [review]

File: src/app/main-page/main-page.component.ts (hunk 1 above)
Comment (aszulc97) [reply src/app/main-page/main-page.component.ts:23]: what a professionalist *clap* *clap* *clap*

Comment (Much4cho) [reply src/app/main-page/main-page.component.ts]: Added a python script that will create us a list of files as a mean-time solution

**Review by aszulc97 — APPROVED** [review]
if it works, it's fine

**Pull request merged**